lib/component.js

```diff
@@ -327,6 +327,15 @@ class Component extends WebComponent {
     }
     this.initializing = true;
 
+    for (const attrsSchemaKey in this._attrsSchema) {
```

Comment:
Usually we'd use Object.keys(); is there a reason to iterate over non-own keys in this case?
```suggestion
    for (const attrsSchemaKey of Object.keys(this._attrsSchema)) {
```

Reply:
I was looking at some microbenchmarks and it showed `for...in` was faster. I'm more than happy to follow whatever the convention is